packages/devp2p/src/rlpx/rlpx.ts

```diff
@@ -233,15 +233,15 @@ export class RLPx extends EventEmitter {
     })
 
     peer.once('close', (reason, disconnectWe) => {
-      if (disconnectWe) {
-        debug(
-          `disconnect from ${socket.remoteAddress}:${socket.remotePort}, reason: ${DISCONNECT_REASONS[reason]}`
-        )
-      } else {
-        debug(
-          `${socket.remoteAddress}:${socket.remotePort} disconnect, reason: ${DISCONNECT_REASONS[reason]}`
-        )
-      }
+      // if (disconnectWe) {
+      //   debug(
+      //     `disconnect from ${socket.remoteAddress}:${socket.remotePort}, reason: ${DISCONNECT_REASONS[reason]}`
+      //   )
+      // } else {
+      //   debug(
+      //     `${socket.remoteAddress}:${socket.remotePort} disconnect, reason: ${DISCONNECT_REASONS[reason]}`
+      //   )
+      // }
```

Comment:
What's this? 😀

Reply:
This seemed entirely reduntant to debug messages coming from peer.ts.  I left it here commented out in case I was mistaken